_source/logzio_collections/_prometheus-sources/telegraf-vsphere.md

```diff
@@ -0,0 +1,126 @@
+If you wish to include or exclude specific metrics, add them to the inputs file as described [here](https://github.com/influxdata/telegraf/blob/release-1.19/plugins/inputs/vsphere/README.md).
+{:.info-box.note}
+<!-- info-box-end -->
+
+
+Here is an example of the configuration file that will enable Telegraf to scrape metrics defined by `vm_metric_include` and `host_metric_include` arrays.
+
+``` ini
+# Read metrics from one or many vCenters
+[[inputs.vsphere]]
+    ## List of vCenter URLs to be monitored. These three lines must be uncommented and edited for the plugin to work.
+  vcenters = [ "https://vcenter.local/sdk" ]
+  username = "user@corp.local"
+  password = "secret"
+
+  ## VMs
+  ## Typical VM metrics (if omitted or empty, all metrics are collected)
+  # vm_include = [ "/*/vm/**"] # Inventory path to VMs to collect (by default all are collected)
+  # vm_exclude = [] # Inventory paths to exclude
+  vm_metric_include = [
+    "cpu.demand.average",
+    "cpu.idle.summation",
+    "cpu.latency.average",
+    "cpu.readiness.average",
+    "cpu.ready.summation",
+    "cpu.run.summation",
+    "cpu.usagemhz.average",
+    "cpu.used.summation",
+    "cpu.wait.summation",
+    "mem.active.average",
+    "mem.granted.average",
+    "mem.latency.average",
+ ]
+
+  ## Hosts
+  ## Typical host metrics (if omitted or empty, all metrics are collected)
+  # host_include = [ "/*/host/**"] # Inventory path to hosts to collect (by default all are collected)
+  # host_exclude [] # Inventory paths to exclude
+  host_metric_include = [
+    "cpu.swapwait.summation",
+    "cpu.usage.average",
+    "cpu.usagemhz.average",
+    "cpu.used.summation",
+    "cpu.utilization.average",
+    "cpu.wait.summation",
+    "disk.deviceReadLatency.average",
+    "disk.deviceWriteLatency.average",
+    "disk.kernelReadLatency.average",
+    "disk.kernelWriteLatency.average",
+    "disk.numberReadAveraged.average",
+    "disk.numberWriteAveraged.average",
+  ]
+
+  ## Datacenters
+  # datacenter_include = [ "/*/host/**"] # Inventory path to clusters to collect (by default all are collected)
+  # datacenter_exclude = [] # Inventory paths to exclude
+  # datacenter_metric_include = [] ## if omitted or empty, all metrics are collected
+  datacenter_metric_exclude = [ "*" ] ## Datacenters are not collected by default.
+  # datacenter_instances = false ## false by default
+  
+```
+
+##### Add the outputs.http plug-in
+
+{% include metric-shipping/telegraf-outputs.md %}
+{% include general-shipping/replace-placeholders-prometheus.html %}
+
+##### Check Logz.io for your metrics
+
+{% include metric-shipping/custom-dashboard.html %} Install the pre-built dashboards to enhance the observability of your metrics.
+
+<!-- logzio-inject:install:grafana:dashboards ids=["VpeHVDlhfo1mF22Lc0UKf"] --> 
+<!-- logzio-inject:install:grafana:dashboards ids=["6CpW1YzdonmTQ8uIXAN5OL"] --> 
+<!-- logzio-inject:install:grafana:dashboards ids=["3AvORCMPVJd8948i9oKaBO"] --> 
+
```

Comment:
The correct way of having more then one dashboard in the install button should be as follows:
`<!-- logzio-inject:install:grafana:dashboards ids=["VpeHVDlhfo1mF22Lc0UKf", "6CpW1YzdonmTQ8uIXAN5OL", "3AvORCMPVJd8948i9oKaBO"] --> `